data/tests/fwupdtool.sh

```diff
@@ -45,6 +51,41 @@ echo "Showing hwids"
 run hwids
 rc=$?; if [ $rc != 0 ]; then error $rc; fi
 
+# ---
+echo "Creating Boot0001"
+run efivar-boot create 0001 Fedora shimx64.efi /tmp/fwupd/mnt
```

Comment:
another test should be added for what happens if entry already exists (IE create it twice)

Reply:
Done